config/storybook-preset-carbon/.babelrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Make sure to include the copyright banner at the top of the file 👀

Reply:
done!